spec/android_spec.rb

```diff
@@ -181,7 +181,7 @@
           out = "package: versionCode='4200066' versionName='4.2.0.66'"
 
           device = DeviceAPI::Android::Device.new(serial: 'SH34RW905290')
-          allow(Open3).to receive(:capture3) { [out, '', $STATUS_ZERO] }
+          allow(Open3).to receive(:capture3) { [out, '', STATUS_OK] }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.